_posts/2019-05-23-quarkus-and-angular-ui-development-mode.adoc

```diff
@@ -0,0 +1,391 @@
+---
+layout: post
+title: Quarkus and Angular UI Development
+date: 2019-05-23 14:00:00.000 -0600
+tags: quarkus ui development
+author: kkhan
+---
+
+== Quarkus and Angular UI Development
+
+In this blog post we will look at how to take advantage of the respective development modes of both Quarkus and Angular CLI and see how we can develop a zero turnaround web application backed by a RESTful API on Quarkus. While I am using Angular, the concepts should be the same for other web application frameworks.
```

Comment:
>  the concepts should be the same for other web application frameworks.

Mention you looked at React and Vue and they support proxying API requests.

So the reader familiar with above two doesn't give up reading when not familiar with Angular